common/services/move.test.js

```diff
@@ -1,10 +1,18 @@
 const proxyquire = require('proxyquire')
 
-const apiClient = require('../lib/api-client')()
+const PersonService = {}
```

Comment:
Perhaps @teneightfive or @solidgoldpig could correct me, but I haven't seen stub test code structured like this, in this particular repository.

I wonder if there's a different way that more closely matches the test structure that is already present. (Or if there is a compelling reason for us to refactor other tests)